src/components/Console.jsx

```diff
@@ -17,11 +19,18 @@ export default function Console({
     return null;
   }
 
+  const mostRecentCompiledProject = compiledProjects.last();
+  const compiledProjectKey = get(mostRecentCompiledProject, 'timestamp', 0);
+
   const console = (
     <div className="console__repl output__item">
       {history.map((entry, key) => (
         // eslint-disable-next-line react/no-array-index-key
-        <ConsoleEntry entry={entry} key={key} />
+        <ConsoleEntry
+          compiledProjectKey={compiledProjectKey}
+          entry={entry}
+          key={key}
```

Comment:
This gave an error  `Do not use Array index in keys  react/no-array-index-key` though not when all on a single line

Reply:
You can use an `eslint-disable-next-line` for that—unfortunately since JS is dynamically typed there’s no way to statically tell the difference between an array iteration and a keyed collection iteration, so eslint produces some false positives.